### DISCUSSION TITLE
ASP.NET Web API

### DISCUSSION THREAD

**Discussion by LeonidasCrypto** [nxrighthere/UnrealCLR#178] [Help] [answered]
I would like to know if I can use UnrealCLR to create a local server using ASP.NET WebApi with NET 5. Thank you!

**Comment by nxrighthere** [comment, marked as answer]
Theoretically, it's possible. You can try to follow [this](https://github.com/dotnet/AspNetCore.Docs/issues/22048) discussion to implement it in a class library. You can use `OnWorldPostBegin()` world event to create a host builder with [`CommandLine`](https://github.com/nxrighthere/UnrealCLR/blob/master/API/CommandLine.md) class for arguments passed to the engine executable.

**Reply by LeonidasCrypto** [reply]
Thanks! I will do it!
